admin-dev/themes/default/template/controllers/login/content.tpl

```diff
@@ -142,7 +142,7 @@
 			</div>
 
 			<div class="front forgot_confirm" style="display: none">
-				<h4 id="forgot_confirm_name">{l s='Please, check your mailbox.' d='Admin.Login.Notification'}<br/><br/>{l s='A link to reset your password has been sent to you.' d='Admin.Login.Notification'}</h4>
+				<h4 id="forgot_confirm_name">{l s='Please, check your mailbox.' d='Admin.Login.Notification'}<br/><br/>{l s='If this email address has been registered in our shop, you will receive a link to reset your password.' d='Admin.Login.Notification'}</h4>
```

Comment:
Is there any way to know in advance if the email matches an account already or not? To know if the wording needs to be updated (eg. "No account was found with this email address.")

ping @PrestaShop/ux-team

Reply:
@l-delin, the point of this change, is to prevent people from knowing this :)

Reply:
Agreed with @kpodemski : we must NOT inform the user, because this would an important information for a hacker